rnd/autogpt_server/autogpt_server/util/service.py

```diff
@@ -108,11 +114,13 @@ def cleanup(self):
 
     @conn_retry
     def __start_pyro(self):
-        daemon = pyro.Daemon(host=pyro_host)
-        ns = pyro.locate_ns(host=pyro_host, port=9090)
-        uri = daemon.register(self)
-        ns.register(self.service_name, uri)
-        logger.info(f"[{self.service_name}] Connected to Pyro; URI = {uri}")
+        host = Config().pyro_host
+        daemon = Pyro5.api.Daemon(host=host, port=self.port)
+        self.uri = daemon.register(self, objectId=self.service_name)
+
+        logger.info(
+            f"Service in start pyro [{self.service_name}] Ready. Object URI = {self.uri}"
```

Comment:
looks like unintentional revert of the log tweak I just merged

Reply:
put it back

Reply:
not yet afaics